lib/resty/core/shdict.lua

```diff
@@ -469,6 +476,18 @@ local function shdict_expire(zone, key, exptime)
     return true
 end
 
```

Comment:
Style: we use 2 blank lines to separate Lua function definitions consistently in this library. Please fix other similar places as well.